cfme/infrastructure/config_management.py

```diff
@@ -181,18 +417,20 @@ def delete(self, cancel=False, wait_deleted=True, force=False):
         """
         if not force and not self.exists:
             return
-        navigate_to(self, 'All')
-        sel.check(Quadicon(self.quad_name, None).checkbox())
+        view = navigate_to(self, 'All')
+        if not view.toolbar.view_selector.is_displayed:
+            view.toolbar.refresh.click()
+        view.toolbar.view_selector.select('List View')
+        row = view.entities.paginator.find_row_on_pages(view.entities.elements,
+                                                        provider_name=self.ui_name)
+        row[0].check()
         item_text = version.pick({'5.6': 'Remove selected items from the VMDB',
```

Comment:
REQUIRED: please remove all '5.6' related stuff. we don't support 5.6 now

Reply:
OK, will do.